### PR TITLE
[032] Update readme to include commands to run test scripts

### PR DESCRIPTION
**Pull request by TaylorJCrawford**

### CODE REVIEW

**Review by siyandlovu1 — COMMENTED** [review]
Able to run the application and test following the steps in the readme

File: README.md [hunk 1]
```diff
@@ -11,3 +11,41 @@ Health Check
...
 
 To see your applications health enter url `http://localhost:8081/healthcheck`
+
+
+ENV Variables
+--- 
+
+To use DropwizardTheITCrowd API you will need to have the following environment variables
+
+- DB_USERNAME &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;# Database Account Username
```
Comment (glenn-hamilton) [inline README.md]: This might be better laid out as a table

File: README.md [hunk 2]
```diff
@@ -11,3 +11,41 @@ Health Check
...
+- DB_PASSWORD &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; # Database Account Password
+- DB_HOST &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; #  Database Hostname 
+- DB_NAME &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; # Database Schema Name
+
+These have been used to connect to the aws database.
+
+        user = System.getenv("DB_USERNAME");
```
Comment (glenn-hamilton) [inline README.md]: It might be better to set this up as `export` commands

File: README.md (hunk 1 above)
Comment (TaylorJCrawford) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (TaylorJCrawford) [reply README.md]: Fixed

**Review by siyandlovu1 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -11,3 +11,47 @@ Health Check
 ---
 
 To see your applications health enter url `http://localhost:8081/healthcheck`
```
Comment (glenn-hamilton) [inline README.md:17]: You've mentioned the healthcheck URL, but not the actual localhost URL (could also add the AWS link as well)

**Review by glenn-hamilton — APPROVED** [review]

**Pull request merged**